src/Resources/views/tailwind_2/_header.html.twig

```diff
@@ -1,21 +1,22 @@
 {# Table Header #}
 {% if table.option('title') or (view is defined and table.option('definition').hasCapability(constant('araise\\CrudBundle\\Enums\\Page::EXPORT'))) or (table.filterExtension and table.filterExtension.filters|length > 0) or (table.searchExtension and table.option('searchable')) %}
-    <header class="whatwedo_table:header sm:flex sm:justify-between mb-4">
+    <header class="whatwedo_table:header sm:flex sm:justify-between my-3">
         <div class="whatwedo_table:header__title flex-1 min-w-0 whatwedo-utility-heading-1 sm:truncate">
             {% if table.option('title') %}
                 <h1>
                     {{ table.option('title')|trans }}
                 </h1>
             {% endif %}
         </div>
-        <div class="mt-4 flex sm:mt-0 sm:ml-4 space-x-2">
+        <div class="mt-4 flex sm:mt-0 sm:ml-4 space-x-3 items-center {{ block is defined ? ' mr-3' : '' }}">
             {% if table.exporters|length > 0 and table.option('definition') and table.option('definition').hasCapability(constant('araise\\CrudBundle\\Enums\\Page::EXPORT')) %}
                 <div
                     {{ stimulus_controller('araise/core-bundle/dropdown') }}
                     class="relative inline-block">
                     <button
                         {{ stimulus_action('araise/core-bundle/dropdown', 'toggle') | stimulus_action('araise/core-bundle/dropdown', 'close', 'keydown.esc') }}
-                        class="whatwedo_table:header__button whatwedo_table:header__button--download whatwedo_table-button h-full" tabindex="-1"
+                        {{ stimulus_controller('araise/crud-bundle/tooltip', { 'title': 'araise_table.download.tooltip'|trans } ) }}
+                        class="whatwedo_table:header__button whatwedo_table:header__button--download whatwedo_table-button" tabindex="-1"
```

Comment:
What's with the funny `tabindex="-1"` here? Are we blocking keyboard and screen reader users from using the tooltip? Why?

Also, let's fix that button too by adding a `sr-only` text into it.